snap/snapcraft.yaml

```
@@ -10,6 +10,7 @@ confinement: classic
 
 environment:
   PATH: "$SNAP/bin:/snap/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
+  LD_LIBRARY_PATH: "$SNAP/none"
```

question: what kind of sorcery is this?

---

I copied this from (I think) Rockcraft.

```suggestion
  # https://github.com/snapcore/snapcraft/issues/4187
  LD_LIBRARY_PATH: "$SNAP/none"
```